REVIEW.md

```diff
@@ -0,0 +1,257 @@
+    resized or optimized in another way?
+
+- [ ] User-facing changes
+    - Does the user interface or other visual elements match existing patterns 
+    in Spyder? If it does not, is there a reason?
+
+Review that addresses these questions on a PR will be prefixed with 
+`[optional]` so that the PR author(s) are clear that this feedback is not a 
+blocker. Addressing this feedback is considered above-and-beyond, and may be 
+more relevant on some PRs than others. PR authors may address these reviews, 
+but are not required to do so in order for the PR to be eligible for merging.
+
+## For documentation/website-focused review
+
+### Required review
+
+The following are areas of review the community expects to be covered in the 
+review of a documentation-focused (sometimes abbreviated as docs) PR. In 
+general, reviewers need to consider whether the content is a good fit, 
+accurate and generally clear to readers, and builds and renders with no 
+simmediate user-visible issues.
+
+**Motivation**
+
+- [ ] Does the change improve the documentation?
+- [ ] Would it help multiple users to have this as a part of the documentation?
+- [ ] Is it appropriate for the documentation and the section it is placed in?
+
+**Content**
+
+- [ ] Is the information accurate? 
+- [ ] Is the change complete, such that it won’t confuse users if it were to 
+appear in the documentation as-is?
+- [ ] Is it clear what the text and images are trying to communicate?
+
+**Images/GIFs**
+
+- [ ] Are screenshots/GIFs reasonably legible and appropriate for the topic?
+- [ ] Are all images/GIFs scaled and optimized? If needed, reviewers should 
+offer to help with this.
+
+**Technical**
+
+- [ ] Do the tests pass in the continuous integration build? If appropriate, 
+help the contributor understand why tests failed.
+- [ ] Does any added/changed content render correctly, without user-visible 
+issues?
+- [ ] Is the content free from typos and unambiguous grammar errors (review 
+the [Style Guide](https://github.com/spyder-ide/spyder-docs/blob/master/STYLEGUIDE.md) 
+for more information)?
+
+Review that addresses these questions on a PR will be prefixed with 
+`[required]` so that the PR author(s) are clear that they are blockers. PR 
+authors are responsible for addressing these reviews if prompted by the 
+reviewer in order for that PR to be eligible for merging.
+
+### Optional review
+
+The following are themes that may be helpful to cover on a 
+documentation-focused PR, but are optional. This may include feedback about 
+non-blocking, substantive, user-visible content improvements to the text and 
+images. 
+
+As with all optional review, please primarily consider how a PR would impact 
+documentation stability, clarity, and quality over aiming for subjective 
+improvements. This section should be used in limited amounts to avoid 
+overtaxing contributors, but can be used judiciously with authors who’ve 
```

Comment:
I'll admit I'm still concerned about creating any of these exception-y kind of loopholes that put contributors in different groups. I want to flag this for any further feedback.

The argument for this split:
Reviewers and contributors who have already been collaborating 
- may not be interested in extra steps of having optional review be limited or live in another issue or PR
- may have higher expectations for themselves and would like to get that feedback directly

The argument against this split:
- Allowing different rules/behavior for different people often leads conflict and abuse of these different rules (intentional or not)
- It's more difficult to hold people accountable when people are not all accountable for the same things when working on the same task.
- It's quite easy to be mistaken about where a contributor might fall in the spectrum of new to returning contributor.

Reply:
It seems that the language should be clarified, and the substantial late-breaking changes that were made just before the PR was submitted arguably shift the meaning somewhat from what I actually intended, but the intent here was _not_ to provide a loophole for reviews of experienced contributors or create a dichotomous split in how PRs were treated, as seems to be the assumption above (that, indeed, is already controlled by the field we agreed to add in the PR template).

Rather, it was aimed at advising _additional_ restraint by reviewers (which is already beyond the level suggested in the equivalent "optional" section in the code-focused review guidelines) at suggesting changes even as "optional" that are within the relatively small number of areas and meet the other limitations mentioned, in cases of newer/less experienced contributors. The intent was to carefully limit requests for anything beyond the minimum required (that regardless of tag, the user may feel some pressure to implement), while still allowing for reviews to judiciously suggest optional improvements when there is a significant potential for net benefit to the project to do on the original PR.

Reply:
I've heavily revised these lines to address this concern and make the intent above much more clear